lib/client/api.go

```diff
@@ -94,6 +94,9 @@ type Config struct {
 	// Labels represent host Labels
 	Labels map[string]string
 
+	// Namespace is nodes namespace
+	Namespace string
```

Comment:
talked about this in person - decided to keep it as is as experimental feature for 1.4. release